src/test/java/uk/gov/hmcts/probate/controller/BusinessValidationUnitTest.java

```diff
@@ -739,7 +742,22 @@ void shouldValidateSolPostCode() {
     }
 
     @Test
-    void shouldValidateSolPostCodeDefaultIht() {
+    void shouldValidateSolPostCodeDefaultIhtSolicitor() {
```

Comment:
similarly, if you added the condition in the rule, you wont need to change these UTs - oly the T for the rule itself